src/components/ui/JobsPageProTipCard.js

```diff
@@ -0,0 +1,65 @@
+// @flow
+/* global React$Node */
+
+import React from 'react'
+
+import theme from '../../styles/theme'
+import typo from '../../styles/typo'
+
+const cmz = require('cmz')
+
+const cx = {
+  card: cmz(
+    typo.regularText,
+    `
+      & {
+        background: ${theme.baseYellow}
+        border: 1px solid ${theme.lineYellow}
+        box-shadow: 4px 4px 0px ${theme.baseYellowDark.fade(0.85)}
+        padding: 24px
+        line-height: 1.4
+        font-weight: 400
+      }
+
+      & a {
+        color: ${theme.typoAnchor}
+      }
+
+      & a:hover {
+        color: ${theme.typoAnchorHover}
+      }
+
+      &:not(:only-child) {
+        margin: 0 0 38px
+      }
+    `
+  ),
+
+  heading: cmz(`
+    & {
+      font-size: 18px
+      line-height: 1.4
+      margin: 0
+    }
+
+    &:not(:only-child) {
+      margin: 0 0 12px
+    }
+  `)
+}
+
+type Props = {
+  children?: React$Node,
+  heading?: string
+}
+
+const JobsPageProTipCard = ({ children, heading }: Props) => children || heading ? (
```

Comment:
I think it should be safe to exclude `children` from the condition check, since if it' falsy it won't be rendered anyways. I can check that yourself.
```suggestion
const JobsPageProTipCard = ({ children, heading }: Props) => heading ? (
```

Reply:
I'm not sure about this change. Because there's no point in showing an empty box:

<img width="504" alt="Screen Shot 2019-11-15 at 11 31 40" src="https://user-images.githubusercontent.com/131859/68950757-8fac9900-079b-11ea-8e5d-1ae274442c82.png">

Reply:
Got it, yeah let's leave as is. BTW, since it's a functional component we can omit `: null` part and just render like that: 

```js
const Component = () => condition && (<div></div>)
```

Reply:
In the past I thought that would work, but it doesn't, here's the output:

<img width="1332" alt="Screen Shot 2019-11-15 at 11 53 17" src="https://user-images.githubusercontent.com/131859/68952444-9f79ac80-079e-11ea-8212-c3863825c15a.png">

Reply:
Ah OK. It works in CodeSandbox: https://codesandbox.io/s/romantic-liskov-84vms